tools/mypy.ini

```diff
@@ -1,4 +1,10 @@
 [mypy]
+# The following are excluded:
+# - tools/runner/ (written in Python 2)
+# - tools/third_party/ (vendored dependencies)
+# - All conftest.py and setup.py files
+# - tests+docs directly under wptrunner+wptserve, which aren't packages.
+exclude = (^tools/runner/|^tools/third_party/|/(conftest|setup)\.py$|^tools/wptrunner/test/|^tools/wptserve/docs/|^tools/wptserve/tests/)
```

Comment:
Can you make the comment clarify when we add things here v. `ignore_errors` below?

Reply:
I've elaborated on the reasoning a fair bit more, PTAL.